README.md

```diff
@@ -19,6 +19,15 @@ Documentation on how to use and extend Ebiten UI is available at [ebitenui.githu
 
 ![Screenshots](ebiten-ui.gif)
 
+Quick Start
+------
+Ebiten UI is written in Go 1.19 which is available at [https://go.dev/](https://go.dev/).
+
+There are Ebiten UI examples that can be found in the `_examples/` folder. Here are some example of how to run them locally:
```

Comment:
This is great, but can you please add a small section about how these commands must be run from the root directory of the project else they won't work?